web/src/components/vis/VisPca.vue

```diff
@@ -174,15 +172,16 @@ export default {
       // Compute the total variance in all the PCs.
       const totVariance = rawPoints.sdev.reduce((acc, x) => acc + x, 0);
       const pctFormat = format('.2%');
-      this.setXLabel(`PC1 (${pctFormat(rawPoints.sdev[0] / totVariance)})`);
-      this.setYLabel(`PC2 (${pctFormat(rawPoints.sdev[1] / totVariance)})`);
```

Comment:
These two `set[X|Y]Label()` calls are needed to update the plot's labels. If we just place `svg` as the first argument to both, will this work as originally intended?

Reply:
I don't think this is correct?  After a transformation, `domain` and `range` may change, so we need to re-run the whole `axisPlot` method (which is done on the next line).  that method calls these two at the end, so these aren't necessary.

Reply:
(We discussed this in person: the solution we reached is to rely on D3's imperative functionality for setting axis labels, and remove the reactive properties from the pipeline altogether.)